Mathlib/GroupTheory/GroupAction/MultiplePrimitivity.lean

```diff
@@ -52,8 +52,8 @@ theorem isPreprimitive_ofFixingSubgroup_conj_iff {s : Set α} {g : G} :
     IsPreprimitive (fixingSubgroup G s) (ofFixingSubgroup G s) ↔
       IsPreprimitive (fixingSubgroup G (g • s)) (ofFixingSubgroup G (g • s)) :=
   isPreprimitive_congr
-    (fixingSubgroupEquivFixingSubgroup G α rfl).surjective
-    (conjMap_ofFixingSubgroup_bijective G α rfl)
+    (fixingSubgroupEquivFixingSubgroup G rfl).surjective
```

Comment:
Is there a reason why the variables in this file are now explicit? It seemed to work fine.

Reply:
A bad merge, certainly. When there are conflicts, it's hard for me to remember which option was the best one.

Reply:
But note : unless I'm mistaken, that's the opposite: the alpha variable is now implicit.